pkg/filter/dt/filter_mysql_query.go

```diff
@@ -144,20 +152,22 @@ func (f *_mysqlFilter) processAfterQueryInsert(ctx context.Context, conn *driver
 
 func (f *_mysqlFilter) processAfterQueryUpdate(ctx context.Context, conn *driver.BackendConnection, updateStmt *ast.UpdateStmt) error {
 	has, xid := misc.HasXIDHint(updateStmt.TableHints)
+	newCtx, span := tracing.GetTraceSpan(ctx, "mysql_process_after_query_update")
```

Comment:
I recommend not adding TraceContext for this location. instead, add inside of func `executor.AfterImage`:
```
newCtx, span := tracing.GetTraceSpan(ctx, "fetch_after_image")
```